packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md

```diff
@@ -100,6 +99,95 @@ To ensure that the Widget renders properly on mobile, include the `viewport` met
+    <link href="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+  </head>
+  <body>
+    <div class="container">
+      <h1 class="text-center">Simple Web Page</h1>
+      <div id="messageBox" class="jumbotron">
+        You are not logged in. Get outta here! Shoo! >:S
+      </div>
+      <!-- where the sign-in form will be displayed -->
+      <div id="okta-login-container"></div>
+      <button id="logout" class="button" onclick="logout()" style="display: none">Logout</button>
+    </div>
+    <script type="text/javascript">
+      var oktaSignIn = new OktaSignIn({
+        baseUrl: "https://${yourOktaDomain}",
+        clientId: "${yourClientId}",
+        authParams: {
+          issuer: "https://${yourOktaDomain}/oauth2/default",
+          responseType: ['token', 'id_token'],
+          display: 'page'
+        }
+      });
+
+      if (oktaSignIn.hasTokensInUrl()) {
+        oktaSignIn.authClient.token.parseFromUrl().then(
+          // If we get here, the user just logged in.
+          function success(res) {
+            var accessToken = res.tokens.accessToken;
+            var idToken = res.tokens.idToken;
+
+            oktaSignIn.authClient.tokenManager.add('accessToken', accessToken);
+            oktaSignIn.authClient.tokenManager.add('idToken', idToken);
+
+            document.getElementById("messageBox").innerHTML = "Hello, " + idToken.claims.email + "! You just logged in! :)";
+            document.getElementById("logout").style.display = 'block';
+          },
+          function error(err) {
+            console.error(err);
+          }
+        );
+      } else {
+        oktaSignIn.authClient.token.getUserInfo().then(function(user) {
+          document.getElementById("messageBox").innerHTML = "Hello, " + user.email + "! You are *still* logged in! :)";
+          document.getElementById("logout").style.display = 'block';
+        }, function(error) {
+          oktaSignIn.renderEl(
+            { el: '#okta-login-container' },
+            function success(res) {},
+            function error(err) {
+              console.error(err);
+            }
+          );
+        });
+      }
+
+      function logout() {
+        oktaSignIn.authClient.signOut();
+        location.reload();
+      }
+    </script>
+  </body>
+</html>
+```
+
+Copy the code above into an `index.html` file on your hard drive.
+
+For this example to work, you'll need to host it on a web server that runs locally on port 8080. To do this, if you have Python 3 installed, you can simply run the command `python -m http.server 8080` in the same directory as `index.html`. If you're on a Mac, you can use `python -m SimpleHTTPServer 8080`. This will open a web server on port 8080 and you'll be able to access your page at http://localhost:8080.
```

Comment:
```suggestion
For this example to work, you'll need to host it on a web server that runs locally on port 8080. If you have Python 3 installed you can simply run the command `python -m http.server 8080` in the same directory as `index.html`. If you're on a Mac, you can use `python -m SimpleHTTPServer 8080`. This will open a web server on port 8080 and you'll be able to access your page at http://localhost:8080.
```